File: COMPUTER_SCIENCE/Basic_Visual_Basic_Basics.vb
Public Class Assingment1_Jake_Tully

    Private Sub btnRed_Click(sender As Object, e As EventArgs) Handles btnRed.Click
        'Changes Form background to red'
        Me.BackColor = Color.Red
    End Sub

    Private Sub btnBlue_Click(sender As Object, e As EventArgs) Handles btnBlue.Click
        'Changes Form background to blue'
        Me.BackColor = Color.Blue
    End Sub

    Private Sub btnDisplay_Click(sender As Object, e As EventArgs) Handles btnDisplay.Click
        'Makes Label visible and sets the text to the contents of the text box'
        lblDisplay.Text = txtDisplay.Text
        lblDisplay.Visible = True

    End Sub

    Private Sub ComboBox1_SelectedIndexChanged(sender As Object, e As EventArgs) Handles BackgroundBox.SelectedIndexChanged
        'Sets background color to the color selected on the combo box'
        If (BackgroundBox.SelectedItem.ToString() = "Pink") Then

            Me.BackColor = Color.Pink

        End If
        If (BackgroundBox.SelectedItem.ToString() = "Yellow") Then

            Me.BackColor = Color.Yellow

        End If
        If (BackgroundBox.SelectedItem.ToString() = "Green") Then

            Me.BackColor = Color.Green

        End If
        If (BackgroundBox.SelectedItem.ToString() = "Orange") Then

            Me.BackColor = Color.Orange

        End If
    End Sub

    Private Sub btnExit_Click(sender As Object, e As EventArgs) Handles btnExit.Click
        'Exits the application'
        Me.Close()
    End Sub

    Private Sub Assingment1_Jake_Tully_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        'Initializes the background color to purple (red+blue)'
        Me.BackColor = Color.Purple
    End Sub
End Class

File: COMPUTER_SCIENCE/Basic_Visual_Basic_Basics.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Assingment1
{
    public partial class Assingment1_jake_tully : Form
    {
        public Assingment1_jake_tully()
        {
            InitializeComponent();
            BackColor = Color.Purple;
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Changes background to blue and foreground to red
            ActiveForm.BackColor = Color.Blue;
        }

        private void button2_Click(object sender, EventArgs e)
        {
            // Changes background to red and foreground to blue
            ActiveForm.BackColor = Color.Red;
        }

        private void btnDisplay_Click(object sender, EventArgs e)
        {
            // Displays text from textbox on in the label
            lblDisplay.Text = txtDisplay.Text;
            lblDisplay.Visible = true;
        }

        private void btnExit_Click(object sender, EventArgs e)
        {
            // closes the application
            Application.Exit();
        }

        private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
        {
          // Sets color to the selected color from drop down menue

            if (comboBox1.SelectedItem.ToString() == "Pink")
            {
                ActiveForm.BackColor = Color.Pink;
            }
            if (comboBox1.SelectedItem.ToString() == "Yellow")
            {
                ActiveForm.BackColor = Color.Yellow;
            }
            if (comboBox1.SelectedItem.ToString() == "Orange")
            {
                ActiveForm.BackColor = Color.Orange;
            }
            if(comboBox1.SelectedItem.ToString() == "Green")
            {
                ActiveForm.BackColor = Color.Green;
            }
        }

        private void Assingment1_jake_tully_Load(object sender, EventArgs e)
        {

        }

        private void button1_Click_1(object sender, EventArgs e)
        {

        }
    }
}
